pkg/cri/resource-manager/cache/container.go

```diff
@@ -228,8 +228,13 @@ func keysInNamespace(m map[string]string, namespace string) []string {
 
 	for key := range m {
 		split := strings.SplitN(key, "/", 2)
-		if len(split) == 2 && split[0] == namespace {
-			keys = append(keys, split[1])
+		if len(split) == 2 {
+			if split[0] == namespace {
+				keys = append(keys, split[1])
+			} else if namespace != "" && strings.HasSuffix(split[0], "."+namespace) {
+				key := strings.TrimSuffix(split[0], "."+namespace) + "/" + split[1]
```

Comment:
> I think this just demonstrates that the "de-facto naming standard" of annotations we follow is kind of inconsistent/flawed.

I still agree, as I already did/told before. Since others don't chime in, I've updated the syntax to be consistent with our expectations. We can bike-shed more about this later if necessary.

Reply:
> If prolly end in trouble if we need to parse annotations in the plain `cri-resource-manager.intel.com` namespace

I didn't quite catch this train of thoughts...

Reply:
I don't have the code, anymore, but my thinking was that if you'd like to get key in the `cri-resource-manager.intel.com` namespace and you'd have an annotations like
- `rdtclass.cri-resource-manager.intel.com/pod=foo`
- `cri-resource-manager.intel.com/bar=baz`

you would have got `rdtclass/pod=foo` and `bar=baz` although you really might've wanted to only have the latter